lodash-plugin/src/test/java/com/github/underscore/lodash/LodashTest.java

```diff
@@ -605,8 +606,12 @@ public void chain() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new String[] {""}).filter(new Predicate<String>() {
             public Boolean apply(String str) { return true; } });
+        $.chain(new String[] {""}).filterIndexed(new PredicateIndexed<String>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.filterIndexed\(PredicateIndexed\) ignored, is this OK in com\.github\.underscore\.lodash\.LodashTest\.chain\(\)